File: arr_sum_of_even_nums/Python/test_sum_of_evens.py
# Testing the sum of evens function 

from sum_of_evens import Solution  
sol = Solution()

some_list = [x for x in range(124, 135)]

def test_sum_of_evens():
    actual = sol.sum_of_evens(some_list)
    expected = 774

    assert actual == expected, f"Test failed! ❌ (as {actual} != {expected}) "
    
    print(f"\nSUCCESS: test_sum_of_evens - Sum of evens in {some_list} is {actual} as expected ({expected}) ✔  ")

File: arr_sum_of_even_nums/JS/test_sum_of_evens.js
// Just test the sum of evens function 

import assert from 'assert/strict';
import { sum_of_evens } from './sum_of_evens.js';

const arr = Array.from({ length: 11 }, (_, i) => 124 + i);

assert.strictEqual(sum_of_evens(arr), 774, 'Failed test cases! ❌');

console.log('All tests passed! ✔  ');
